drift/lib/src/runtime/manager/manager.dart

```diff
@@ -505,12 +754,35 @@ class ProcessedTableManager<
         $FilterComposer extends FilterComposer<$Database, $Table>,
         $OrderingComposer extends OrderingComposer<$Database, $Table>,
         $CreateCompanionCallback extends Function,
-        $UpdateCompanionCallback extends Function>
-    extends BaseTableManager<$Database, $Table, $Dataclass, $FilterComposer,
-        $OrderingComposer, $CreateCompanionCallback, $UpdateCompanionCallback> {
+        $UpdateCompanionCallback extends Function,
+        $DataclassWithReferences,
+        $ActiveDataclass,
+        $CreatePrefetchHooksCallback extends Function>
+    extends BaseTableManager<
+        $Database,
+        $Table,
+        $Dataclass,
+        $FilterComposer,
+        $OrderingComposer,
+        $CreateCompanionCallback,
+        $UpdateCompanionCallback,
+        $DataclassWithReferences,
+        $ActiveDataclass,
+        $CreatePrefetchHooksCallback> {
   /// Create a new [ProcessedTableManager] instance
   @internal
   ProcessedTableManager(super.$state);
+
+  /// The prefetched data for this manager
+  /// This will be null if the manager was not created with prefetched data
```

Comment:
Maybe it's simpler and more precise to say "Prefetched data, if references with prefetching enabled were added to this manager".